compiler/hsSyn/HsTypes.hs

```diff
@@ -466,11 +466,15 @@ instance OutputableBndr HsIPName where
 --------------------------------------------------
 
 -- | Haskell Type Variable Binder
+data HasBraces = Yes | No
```

Comment:
Since constructors are not namespaced, I think it would be more appropriate to use names like this:

```
data TvbBraces = TvbHasBraces | TvbNoBraces
```